R/completion.R

```diff
@@ -0,0 +1,22 @@
+install_github_completer <- function(env) {
+  if (completeme::current_function(env) == "install_github" && completeme::inside_quotes(env)) {
```

Comment:
An early return would make this code a bit simpler. Might be worth providing completeme helper? Should it also check the argument name?